.circleci/common.sh

```diff
@@ -156,8 +156,13 @@ function run_torch_xla_tests() {
     fi
 
     pushd test/cpp
-    echo "Running C++ Tests"
+    echo "Running C++ Tests on XRT"
     ./run_tests.sh
+    # TODO(wcromar): Enable PJRT C++ tests on GPU
+    if [ -z "$GPU_NUM_DEVICES" ]; then
+      echo "Running C++ Tests on PJRT"
+      PJRT_DEVICE=CPU ./run_tests.sh
```

Comment:
wouldn't this double our CI test runtime?

Reply:
I'm skipping it on GPU, which is the much longer test suite, so the total time for CI results will not increase. This is also only adding the C++ tests for PJRT, which account for a small portion of the total runtime of the CPU tests. IME the python tests take much longer because there are more of them.

Reply:
oh ok I missed that this only runs CPP test.